low-level/files/opt/seagate/sspl/setup/sspl_setup_init.py

```diff
@@ -0,0 +1,143 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+#################################################################
+# This script performs following operations.
+# - Creates datapath as defined in /etc/sspl.conf
+# - Check dependencies for roles other than '<product>'
+#################################################################
+
+import os
+import pwd
+import errno
+
+from cortx.sspl.bin.sspl_constants import file_store_config_path
+from cortx.utils.validator.v_service import ServiceV
+from cortx.utils.validator.v_pkg import PkgV
+from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.error import SetupError
+from cortx.utils.conf_store import Conf
+
+class SSPLInit:
+
+    """Init Setup Interface"""
+
+    name = "init"
+
+    SSU_DEPENDENCY_RPMS = [
+                "sg3_utils",
+                "gemhpi",
+                "pull_sea_logs",
+                "python-hpi",
+                "zabbix-agent-lib",
+                "zabbix-api-gescheit",
+                "zabbix-xrtx-lib",
+                "python-openhpi-baselib",
+                "zabbix-collector",
+    ]
+
+    SSU_REQUIRED_PROCESSES = [
+                    "openhpid",
+                    "dcs-collectord",
+    ]
+
+    VM_DEPENDENCY_RPMS = []
+
+    HPI_PATH = '/tmp/dcs/hpi'
+    MDADM_PATH = '/etc/mdadm.conf'
```

Comment:
Should I move these in  sspl_constants.py??

Reply:
Yes.